openstf/model/serializer.py

```diff
@@ -144,6 +154,16 @@ def load_model_from_path(self, model_path):
 
         return loaded_model
 
+    def determine_model_age_from_pid(self, pid) -> float:
```

Comment:
Great function, nicely implemented. One minor thing: could you add a typehint for the pid argument? (self, pid: int)
Also the argument is missing in the docstring.